<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="Title">
       <property name="font">
        <font>
         <family>Caladea</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Dialog Tab</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="port">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>16</number>
         </property>
         <property name="maxVisibleItems">
          <number>20</number>
         </property>
         <item>
          <property name="text">
           <string>Port 0x0 Console</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0x1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0x2 Parameters</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0x3 Commander</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0x4 Memory</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0x5 Log</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0x6 Localization</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0x7 Generic set points</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0x8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0x9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0xA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0xB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0xC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0xD Platform</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0xE Client-side debugging</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Port 0xF Link layer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>All ports</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string> Channel: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="channel">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentText">
          <string>All</string>
         </property>
         <property name="currentIndex">
          <number>8</number>
         </property>
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>All</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Length of text: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="line_number">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>80</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>non limit√©</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="direction">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>In</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Out</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Both</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>350</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <italic>true</italic>
        </font>
       </property>
       <property name="title">
        <string>Hear</string>
       </property>
       <widget class="QPlainTextEdit" name="receive">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>731</width>
          <height>321</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>180</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <italic>true</italic>
        </font>
       </property>
       <property name="title">
        <string>Speak</string>
       </property>
       <widget class="QTextEdit" name="send">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>731</width>
          <height>151</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
